assets/js/modules/tagmanager/components/settings/SettingsForm.js

```diff
@@ -29,8 +35,33 @@ import {
 } from '../common';
 import StoreErrorNotices from '../../../../components/StoreErrorNotices';
 import { MODULES_TAGMANAGER } from '../../datastore/constants';
+import ProgressBar from '../../../../components/ProgressBar';
+const { useSelect } = Data;
 
 export default function SettingsForm() {
+	const accountID = useSelect( ( select ) =>
+		select( MODULES_TAGMANAGER ).getAccountID()
+	);
+	const internalContainerID = useSelect( ( select ) =>
+		select( MODULES_TAGMANAGER ).getInternalContainerID()
+	);
+	const internalAMPContainerID = useSelect( ( select ) =>
+		select( MODULES_TAGMANAGER ).getInternalAMPContainerID()
+	);
+	const isResolvingWebGetLiveContainerVersion = useSelect( ( select ) =>
+		select( MODULES_TAGMANAGER ).isResolving( 'getLiveContainerVersion', [
+			accountID,
+			internalContainerID,
+		] )
+	);
+	const isResolvingAMPGetLiveContainerVersion = useSelect(
+		( select ) =>
+			select( MODULES_TAGMANAGER ).isResolving(
+				'getLiveContainerVersion'
+			),
+		[ accountID, internalAMPContainerID ]
+	);
```

Comment:
Let's simplify this into a single `useSelect` for a variable like `isResolvingLiveContainerVersion` which would return essentially `isResolvingWebGetLiveContainerVersion || 			isResolvingAMPGetLiveContainerVersion`

All of the other values can be selected within as they're only needed for deriving the one value. Our convention for `useSelect` is to always return a single value but it's fine to select other things as needed within the select function 👍